### PR TITLE
Add Vincent Lee

### PR DESCRIPTION
**Pull request by drawntotheflame**

### CODE REVIEW

**Comment by drawntotheflame** [comment]
@zzorba Please can you double-check this for me, then I'll nudge Kam that it can go live. Thank you.

**Comment by 5argon** [comment]
Missing just the `,` character on line 123!

https://github.com/Kamalisk/arkhamdb-json-data/blob/c0b3de0dd476fb5cfbc3ab77ca9934111b65c1a7/pack/tsk/tskp.json#L123

**Comment by elkeinkrad** [comment]
I found that this regex cannot search Cheat Death

> Cheat Death
> 
> Fast. Play when you would be defeated.
> Disengage from each enemy engaged with you, discard all cards in your threat area, **heal 2 horror and 2 damage**, and move to any revealed location with no enemies. If it is your turn, end your turn. Remove Cheat Death from the game.

I think that this expression is irregular and those expression will not be released more, so that we only considered cheat death case for "horror", in my opinion.

Thus, in current released cards, it's ok to,
`[Hh]eals? (that much )?(2 horror and )?((\\d+|all|(X total)) )?damage`
I just add "(2 horror and )?" in current regular expression

In fact, "that much" for Ancient Stone and "X total" for Rite of Equilibrium are not necessary yet. Thus, the following is also ok.
`[Hh]eals? (2 horror and )?((\\d+|all) )?damage`

For more robust,
`[Hh]eals? (that much )?(\\d+ horror (and|or) )?(\\d+|all|(X total) )?damage`

**Comment by drawntotheflame** [comment]
thanks @5argon and @elkeinkrad :D

File: pack/tsk/tskp.json [hunk 1]
```diff
@@ -29,7 +29,98 @@
...
+        "back_text": "<b>Deck Size</b>: 30.\n<b>Deckbuilding Options</b>: Seeker cards ([seeker]) level 0-3, Neutral cards level 0-5, cards that \"heal damage\" level 0-5, up to 15 other Guardian and/or Survivor cards level 0-1 ([guardian] and/or [survivor]).\n<b>Deckbuilding Requirements</b> (do not count toward deck size): Bonesaw, 1 [per_investigator] copies of On the Mend, Wounded Bystander, 1 random basic weakness.\n<b>Additional Setup</b>: You begin each game with 1 [per_investigator] copies of On the Mend set aside, out of play.",
+        "code": "09004",
+        "deck_limit": 1,
+        "deck_options": [
+            { "faction": ["seeker"], "level": { "min": 0, "max": 3 } },
+            { "faction": ["neutral"], "level": { "min": 0, "max": 5 } },
+            {
+                "text": [
+                    "[Hh]eals? (that much )?(\\d+ horror (and|or) )?(\\d+|all|(X total) )?damage"
```
Comment (zzorba) [inline pack/tsk/tskp.json:42]: I think we should consider just using the flipped Carolyn.  As far as I can tell, this grabs all the cards:

```
First Aid (0)
Medical Texts (0)
First Aid (0)
Medical Texts (0)
First Aid (3)
Emergency Aid (0)
Painkillers (0)
Strange Solution - Restorative Concoction (4)
Inspiring Presence (0)
First Aid (3)
Cheat Death (5)
Second Wind (0)
Thermos (0)
Deny Existence (5)
Jessica Hyde - Wrong Place, Wrong Time (1)
Healing Words (0)
The Black Cat - A Liar, or a Prophet, or Both (5)
Spiritual Resolve (5)
Manipulate Destiny (2)
Spirit of Humanity (2)
Purifying Corruption (4)
Medical Texts (2)
Archive of Conduits - Gateway to Paradise (4)
Meditative Trance (0)
Healing Words (3)
Bandages (0)
Medical Student (0)
Precious Memento - From a Former Life (4)
Precious Memento - From a Future Life (4)
Earthly Serenity (1)
Earthly Serenity (4)
Call for Backup (2)
Hallowed Chalice (0)
Lonnie Ritter - Feisty Mechanic (0)
```

```suggestion
                    "[Hh]eals? (that much )?((((\\d+)|(all)|(X total)) )?horror (from that asset )?(and|or) )?(((\\d+)|(all)|(X total)) )?damage"
```

File: pack/tsk/tskp.json [hunk 2]
```diff
@@ -29,7 +29,98 @@
...
+        "faction_code": "neutral",
+        "illustrator": "JB Casacop",
+        "name": "Bonesaw",
+        "pack_code": "tskp",
+        "position": 5,
+        "quantity": 1,
+        "skill_combat": 1,
+        "skill_intellect": 1,
+        "skill_wild": 1,
+        "restrictions": "investigator:09004",
```
Comment (zzorba) [inline pack/tsk/tskp.json:87]: Missing `"slot": "Hand",`

**Review by zzorba — COMMENTED** [review]
Looks like Bonesaw is missing the hand slot, but otherwise LGTM

**Comment by drawntotheflame** [comment]
thanks @zzorba . Over to you, @Kamalisk

**Pull request merged**